script/script.js

```diff
@@ -0,0 +1,326 @@
+        //clean up & add this move to the player's array
+        let item = event.target;
+        item = item.outerHTML.toString();
+        item = item.slice(9, 14);
+        xMoves.push(item);
+        //add it to the total moves
+        totalMoves.push(event.target);
+        //check to see if there is a winning combo
+        checkWin();
+        //trigger computer move
+        setTimeout(function () {
+          gameBoard.computerMove(event);
+        }, 1500);
+      }
+      //ask player to play somewhere else if the cell is not empty
+      else alert("You can't play there!");
+    } else if (currentPlayer === "O") {
+      //choose a random number between 0 and 8
+      let cellNum = Math.floor(Math.random() * 9);
+      //turn this into an index to randomly select a cell for computer to play in
+      let compChoice = allCells[cellNum];
+      //only fill in the cell if it is "false" because it's empty
+      if (!compChoice.textContent) {
+        compChoice.textContent = "O";
+        //switch players
+        currentPlayer = "X";
+        playerTurn.innerHTML = "It's " + player1.value + "'s turn";
+        //clean up & add this move to the player's array
+        let item = compChoice;
+        item = item.outerHTML.toString();
+        item = item.slice(9, 14);
+        oMoves.push(item);
+        //add it to the total moves
+        totalMoves.push(event.target);
+        //check to see if there is a winning combo
+        checkWin();
+        //if there is already something in the cell, trigger the computer choice function again
+      } else if (compChoice.textContent) {
+        gameBoard.computerMove(event);
+      }
+    }
+  },
+};
+
+//all the winning combos
+let winningArrays = [
+  ["cell1", "cell2", "cell3"],
+  ["cell4", "cell5", "cell6"],
+  ["cell7", "cell8", "cell9"],
+  ["cell1", "cell4", "cell7"],
+  ["cell2", "cell5", "cell8"],
+  ["cell3", "cell6", "cell9"],
+  ["cell1", "cell5", "cell9"],
+  ["cell3", "cell5", "cell7"],
+];
+
+function checkWin() {
+  //check to see if the player has a winning combination - if they do, then highlight the winning cells, set an alert, and stop the timer
+  //loop through the arrays
+  for (let i = 0; i < winningArrays.length; i++) {
+    //check to see if array of player X's moves has a winning array by looping through each sub-array
+    if (
+      xMoves.includes(winningArrays[i][0].toString()) &&
+      xMoves.includes(winningArrays[i][1].toString()) &&
+      xMoves.includes(winningArrays[i][2].toString())
+    ) {
+      //identify which were the winning cells
+      let winningCell1 = winningArrays[i][0];
+      let winningCell2 = winningArrays[i][1];
+      let winningCell3 = winningArrays[i][2];
+      //change color of winning cells
+      cellKey[winningCell1].style.backgroundColor = "red";
+      cellKey[winningCell2].style.backgroundColor = "red";
+      cellKey[winningCell3].style.backgroundColor = "red";
```

Comment:
One thing I did notice is this block of functionality is repeated a few times (to check for different win scenarios, apply borders, etc..) which is totally fine! However, I wonder if there are any easier ways to check for game wins and so on that might take less code? 
The reason I bring this up is if you find yourself re-typing code in a program, there is usually an easier way to get the same logic/result/answer.
Something to think about if you ever re-visit this project!